indico/web/client/js/react/components/ICSCalendarLink.jsx

```diff
@@ -154,6 +161,20 @@ export default function ICSCalendarLink({
             </Grid.Row>
           </Grid>
         )}
+        <strong styleName="export-option">Download</strong>
+        <div styleName="download-option">
+          <span>
+            <Translate>
+              Download an iCalendar file that you can use in calendaring applications.
+            </Translate>
+          </span>
+          <Button
+            styleName="download-button"
+            as="a"
+            href={option ? option.url : ''}
```

Comment:
I guess this is missing while still retrieving the URL? If yes, you may want to add `disabled={!option}`

Reply:
I thought about disabling the button, but [line 153](https://github.com/javfg/indico/blob/ical-downloadbutton/indico/web/client/js/react/components/ICSCalendarLink.jsx#L153) hides it while loads, so discarded the idea in the end.

Reply:
Sure, but that's only hiding the copy button, not the download button...

Reply:
Aaah, I see the problem. There was another thing going on: [`setOpen`](https://github.com/javfg/indico/blob/ical-downloadbutton/indico/web/client/js/react/components/ICSCalendarLink.jsx#L63) was being run after the `await`, so I had no way to check what was happening in the form before the URL loaded. It just did not show until `option.url` was there.

Reply:
I'll fix that.